src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  padding: 0;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  background-color: rgb(235, 235, 235);
+
+  height: 5000px;
+}
+
+.header {
+  position: fixed;
+  width: 100vw;
+  background-color: #fff;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  max-width: 1100px;
```

Comment:
no, it shouldn't be like that, that's why you didn't pass the test
```suggestion
```

Reply:
Okay, I thought the task specified that the content should not be discharged at a width of 2000px+
Now I remove Max-Width ;)